### ISSUE TITLE
tests: stop using ubi8 (this is the "0x3c" flake)

### ISSUE THREAD

**Issue by edsantiago** [containers/buildah#4318]
Use of `ubi8` images is causing one of our big flakes, mostly in podman (because it runs many more CI jobs than buildah) but also in buildah CI:
```
$ buildah build -f Containerfile --pull=false -q --arch=amd64 -t image-amd --signature-policy /var/tmp/go/src/github.com/containers/buildah/tests/./policy.json /var/tmp/buildah_tests.sufdm8/my-dir
error creating build container: copying system image from manifest list:
     reading signatures:
     parsing signature https://access.redhat.com/webassets/docker/content/sigstore/ubi8-minimal@sha256=SHA/signature-7:
     unrecognized signature format, starting with binary 0x3c
```
I think this is also the cause of the horrid `multiple-platform-no-run` flake, or at least one of the causes.

We can't rely on flaky registries. Can all tests that use `ubi8-*`  be fixed to use a different image?

### [Integration] 295 bud --pull=false --arch test
* [unknown image] : Integration fedora-35 w/ vfs
  * PR #4111
    * [07-14-2022 12:16](https://api.cirrus-ci.com/v1/task/5063862180380672/logs/integration_test.log)
* [unknown image] : Integration rootless fedora-35 w/ overlay
  * PR #4111
    * [07-14-2022 12:19](https://api.cirrus-ci.com/v1/task/5070459250147328/logs/integration_test.log)
* [unknown image] : Integration rootless ubuntu-2204 w/ overlay
  * PR #4111
    * [07-14-2022 12:07](https://api.cirrus-ci.com/v1/task/6196359156989952/logs/integration_test.log)

### [Containerized] 295 bud --pull=false --arch test
* [unknown image] : Containerized Integration
  * PR #4111
    * [07-14-2022 12:04](https://api.cirrus-ci.com/v1/task/4788984273436672/logs/integration_test.log)

...and, in podman-land:

### [bud] 277 bud --pull=false --arch test
* fedora-36 : bud podman fedora-36 root host
  * PR #16050
    * [10-04 17:51](https://api.cirrus-ci.com/v1/artifact/task/4683925472673792/html/bud-podman-fedora-36-root-host.log.html#t--00277)
* fedora-36 : bud remote fedora-36 root host [remote]
  * PR #16050
    * [10-04 17:52](https://api.cirrus-ci.com/v1/artifact/task/5809825379516416/html/bud-remote-fedora-36-root-host.log.html#t--00277)

### [bud] 268 bud --pull=false --arch test
* fedora-36 : bud podman fedora-36 root host
  * PR #15593
    * [09-06-2022 12:14](https://api.cirrus-ci.com/v1/artifact/task/5420634996998144/html/bud-podman-fedora-36-root-host.log.html#t--00268)
  * PR #15285
    * [08-11-2022 09:02](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/6068841762521088/html/bud-podman-fedora-36-root-host.log.html#t--00268)
* fedora-36 : bud remote fedora-36 root host [remote]
  * PR #15593
    * [09-06-2022 12:14](https://api.cirrus-ci.com/v1/artifact/task/6546534903840768/html/bud-remote-fedora-36-root-host.log.html#t--00268)
  * PR #15285
    * [08-11-2022 08:59](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/5505891809099776/html/bud-remote-fedora-36-root-host.log.html#t--00268)

### [bud] 261 bud --pull=false --arch test
* fedora-36 : bud podman fedora-36 root host
  * PR #15192
    * [08-04-2022 08:57](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/6541779351633920/html/bud-podman-fedora-36-root-host.log.html#t--00261)
* fedora-36 : bud remote fedora-36 root host [remote]
  * PR #15192
    * [08-04-2022 09:01](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/4712192003014656/html/bud-remote-fedora-36-root-host.log.html#t--00261)

**Comment by edsantiago** [comment]
[Every](https://api.cirrus-ci.com/v1/task/6360653660160000/logs/integration_test.log). [Single](https://api.cirrus-ci.com/v1/task/6079178683449344/logs/integration_test.log). [Integration](https://api.cirrus-ci.com/v1/task/6642128636870656/logs/integration_test.log). [Job](https://api.cirrus-ci.com/v1/task/4601435055718400/logs/integration_test.log). [In](https://api.cirrus-ci.com/v1/task/5516228730028032/logs/integration_test.log). [My](https://api.cirrus-ci.com/v1/task/4953278776606720/logs/integration_test.log). [PR](https://api.cirrus-ci.com/v1/task/5727334962561024/logs/integration_test.log).

**Comment by mtrmac** [comment]
For the record: This is the signature server serving 200 OK with HTML content; it is supposed to serve 200 OK with a signature, or 404 — or some non-200 error at least.

**Comment by edsantiago** [comment]
All CI (buildah and podman), all gating tests, are now broken because ubi8 changed something.
```
✗ bud --pull=false --arch test
   #/vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
   #|     FAIL: buildah inspect --format {{ index .Docker.Config.Labels "architecture" }} image-arm
   #| expected: =~ 'arm64'
   #|   actual:    'aarch64'
   #\^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
When  we use images we don't control, on registries we don't control, bad things happen.

**Comment by edsantiago** [comment]
Fixed in #4384 and #4397.
